src/components/Shared/ClearGridPhyConfirmation.js

```diff
@@ -0,0 +1,21 @@
+import { ControlContext } from 'src/providers/ControlContext'
+import ConfirmationDialog from '../ConfirmationDialog'
+import { useContext } from 'react'
+
+const ClearGridPhyConfirmation = ({ window, onConfirm, fullScreen, open }) => {
```

Comment:
Can we rename this component to make it more general, so it can be reused on other screens in the future?